src/lib/phdi-building-blocks/phdi_building_blocks/standardize.py

```diff
@@ -59,18 +62,139 @@ def standardize_patient_name(
     return bundle
 
 
-def standardize_phone(raw: str) -> str:
-    """Make sure it's 10 digits, remove everything else
+def country_extractor(
+    resource: dict, code_type: Literal["alpha_2", "alpha_3", "numeric"] = "alpha_2"
+) -> Generator[str, None, None]:
+    """
+    Given a FHIR resource yield a generator containing all of the counries found in the
+    resource in a standard form sepcified by code_type.
+
+    Args:
+        resource (dict): A FHIR resource respresented as a dictionary.
+        code_type (str): A string equal to 'alpha_2', 'alpha_3', or 'numeric' to specify
+                         which type of standard country identifier to generate.
+    Returns:
+        (generator): A generator to return standardized countries when it is iterated
+                     over.
+
+    We currently only handle extracting countries from patient resources, however as
+    need arises is the future funcionality can be expanded.
+    """
+    resource_type = resource.get("resource").get("resourceType")
+    if resource_type == "Patient":
+        for address in resource.get("resource").get("address"):
+            country = address.get("country")
+            yield standardize_country(country, code_type)
+
 
-    >>> standardize_phone("(555) 555-1212")
-    '5555551212'
+def standardize_phone(
+    raw: str, resource: dict = None, country_extractor: Callable = country_extractor
```

Comment:
I'm actually working on cleaning this up now Spencer

Reply:
[59b862b](https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/123/commits/59b862b89d61b1008946240041079380c18f1fc1) replaces the fuzzy matching with explicit searches on `name `and `official name`. This will let us make more than a good faith effort to handle cases where the FHIR bundle doesn't have an ISO country code, but avoids fuzzy matching.